spec/helpers/navbar_helper_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe NavbarHelper do
+  context "when the settings are 'on' to displaying the navbar" do
+    let(:settings) { true }
+
+    it "should return true for the pages where the navbar can be hidden" do
+      subject.hidden_places.each do |controller, actions|
+        actions.each { |action| expect(subject.show_navbar?(controller, action, settings)).to eq(true) }
+      end
+    end
+
+    it "should return true for the pages not concerned by the displaying option" do
+      expect(subject.show_navbar?('always_display', 'the_navbar', settings))
```

Comment:
It might be good here to display data that's more realistic so people can read the test to know what our goals are. For example, setting the first two arguments to `fake_test_controller` and `index` might be a better example. Or maybe `always_displayed_controller`? Whatever you'd like

Reply:
Done :+1: